src/modules/username.rs

```diff
@@ -2,22 +2,15 @@ use super::{Context, Module, ModuleConfig};
 
 use crate::configs::username::UsernameConfig;
 use crate::formatter::StringFormatter;
+#[cfg(not(test))]
+use whoami::username;
 
-#[cfg(not(target_os = "windows"))]
-const USERNAME_ENV_VAR: &str = "USER";
-
-#[cfg(target_os = "windows")]
-const USERNAME_ENV_VAR: &str = "USERNAME";
+#[cfg(test)]
+use tests::username;
```

Comment:
I would prefer handling this with a small wrapper function, and it seems like a good idea to keep around the possibility of overriding the username in the tests:
```rust
fn username() -> String {
    #[cfg(test)]
    if let Some(user) = context.get_env("USER") {
        return user;
    };
    whoami::username()
}

Reply:
Thanks for your suggest. I have  optimized the codes, and review again please.